### PR TITLE
Require other css moudles

### PR DESCRIPTION
**Pull request by DiscordCSS**
:^)

### CODE REVIEW

File: bot.css [hunk 1]
```diff
@@ -1,4 +1,6 @@
 bot {
+ require: 'discord.css';
```
Comment (DiscordCSS) [inline bot.css]: requireing the progarm

File: bot.css [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
+ require: 'pineapplePicApi';
```
Comment (DiscordCSS) [inline bot.css]: require another api

**Comment by Dev1nee** [comment]
unneeded

**Comment by Dev1nee** [comment]
already handled with the latest update

**Pull request closed without merging**